Resources/Textures/Objects/Fun/spraycans.rsi/meta.json

```diff
@@ -70,6 +79,22 @@
     {
       "name": "equipped-BELT",
       "directions": 4
+    },
+    {
+      "name": "clown-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "clown-inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "spray-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "spray-inhand-left",
+      "directions": 4
     }
   ]
 }
```

Comment:
You can also paint a holographic signs from RMB, lol
![image](https://github.com/space-wizards/space-station-14/assets/96445749/303805b4-ff44-4d8a-90dd-e622a983d975)
also you can paint Singula and tesla (its fun)

It might make sense to make a "can be painted" component, and manually add it to the base entities to avoid this sort of thing.

Reply:
Will probably need something like a canpaintcomponent to resolve certain edge cases or a cannotpaintcomponent whichever is easier.